test/integration/integration-tests.js

```diff
@@ -181,6 +182,13 @@ test('Gets space for read only tests', (t) => {
       assetReadOnlyTests(t, space)
     })
 })
+test('Gets v2 space for read only tests', (t) => {
+  return v2Client.getSpace('w6xueg32zr68')
+    .then(space => {
+      userTests(t, space)
```

Comment:
I am not sure why do you need `userTests` apart of the `spaceMembersTests` ?

Reply:
oops, I wanted to do a separate test suite for userTests and spaceMembers and leave the other one only for environment tests since sometimes they were failing when combined, so to keep them less flaky; i will remove lines 203 and 204